lib/core/src/lib/auth/services/oidc-authentication.service.ts

```diff
@@ -138,8 +138,8 @@ export class OidcAuthenticationService extends BaseAuthenticationService {
         return this.getUsername();
     }
 
-    ssoImplicitLogin() {
-        this.auth.login();
+    ssoLogin(redirectUrl?: string) {
+        this.auth.login(redirectUrl);
```

Comment:
I just wonder, you mark `redirectUrl` as an optional, but you pass it to login at the same time, is it gonna work when `redirectUrl` wont be provided?

Reply:
yes: 
abstract login(currentUrl?: string): Promise<void> | void;